LICENSE-MIT.md

```diff
@@ -0,0 +1,7 @@
+Copyright 2020 Daniel Gee
```

Comment:
To avoid questions of "who should go here" on multi-contributor projects I've started skipping the copyright notice since it's not necessary.